chainio/clients/wallet/fireblocks_wallet_test.go

```diff
@@ -323,7 +323,30 @@ func TestWaitForTransactionReceiptFailFromFireblocks(t *testing.T) {
 	}, nil)
 
 	receipt, err := sender.GetTransactionReceipt(context.Background(), expectedTxHash)
-	assert.Error(t, err)
+	assert.ErrorAs(t, err, &wallet.ErrReceiptNotYetAvailable)
+	assert.Nil(t, receipt)
+}
+
+func TestWaitForTransactionReceiptStuckAtFireblocks(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	fireblocksClient := cmocks.NewMockFireblocksClient(ctrl)
+	ethClient := mocks.NewMockEthClient(ctrl)
+	logger, err := logging.NewZapLogger(logging.Development)
```

Comment:
just a nit as this is just a test, but let's try to only use slog moving forward, to be consistent inside the sdk.

Reply:
Agree we should try to use one type of logger in the codebase. Are we going with slog? I don't see any slog usage in this repo, but do see zap logger being used